llvm/tools/llvm-ir2vec/llvm-ir2vec.cpp

```diff
@@ -216,56 +222,139 @@ class IR2VecTool {
         unsigned Opcode = Vocabulary::getIndex(I.getOpcode());
         unsigned TypeID = Vocabulary::getIndex(I.getType()->getTypeID());
 
-        // Add "Next" relationship with previous instruction
         if (HasPrevOpcode) {
```

Comment:
Please retain the comments here and below. They can be removed/cleaned up if they are not appropriate.